### DISCUSSION TITLE
BIRT Docker Container

### DISCUSSION THREAD

**Discussion by viniciusalvess** [eclipse-birt/birt#1569] [General]
Hi everyone,

I created this repo that runs birt runtime on a tomcat docker container configured with a jndi datasource out of the box.
Feel free to test it out and let me know if there is something I can do to improve it.

https://github.com/viniciusalvess/birt-docker-container/tree/main

**Comment by speckyspooky** [comment]
Very nice, we are planning during the year a familiar reserach to create a container
with BIRT and our framework so it will help us.

Only thing we will use WildFly and Kubernetes bute we will take a deep look into your setup.

**Reply by viniciusalvess** [reply]
I believe we'll be able to have Tomcat and Wildfly on the same project, we'll have to add a separate Dockerfile file for Wildfly though.
Today I released a new version, adjusting the project structure and fixing some bugs I had when I deployed it yesterday to my production environment. 
Yeah please go ahead and check the project out  and let me know if there is anything I can do to help with your Wildfly setup.

**Reply by danfoxley** [reply]
> Very nice, we are planning during the year a familiar reserach to create a container with BIRT and our framework so it will help us.

Hi,
Please exuse the barge into this thread. Was any research done for BIRT as a container? Any reason not to try or use @viniciusalvess solution?

Thanks,
Dan Foxley
